src/Microsoft.Framework.PackageManager/NuGet/Core/Configuration/Settings.cs

```diff
@@ -172,15 +169,11 @@ public string ConfigFilePath
             if (configFileName == null)
             {
                 // load %AppData%\NuGet\NuGet.config
-#if DNX451
-                var appDataPath = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
-#else
-                var appDataPath = Environment.GetEnvironmentVariable("APPDATA");
-#endif
-                if (!String.IsNullOrEmpty(appDataPath))
+                var userSettingsDir = DnuEnvironment.GetFolderPath(DnuFolderPath.UserSettingsDirectory);
```

Comment:
This doesn't look right. Why isn't it directory returning the NuGet/Config folder.